src/components/SearchResultDetail/SearchResultDetail.tsx

```diff
@@ -32,6 +58,26 @@ const SearchResultDetail = ({ id }: { id: string }) => {
     findSearchResult();
   }, [id]);
 
+  const downloadPdf = async () => {
+    try {
+      setDownloadLoading(true);
+      const pdfString = await exportSearchResultPdfAction(searchResult.id);
+      if (!pdfString) {
+        return toast('Can not download the pdf version', 'error');
```

Comment:
This error message is still not consistent with the rest of the codebase:

 ```suggestion
        return toast('The PDF file cannot be exported', 'error');
```

How about rasing an error so that the code flows to the `catch` block (thus avoiding to use the same error message twice).